easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -645,6 +660,16 @@ def sanity_check_step(self, *args, **kwargs):
                 exts_filter = (orig_exts_filter[0].replace('python', self.python_cmd), orig_exts_filter[1])
                 kwargs.update({'exts_filter': exts_filter})
 
+        if self.cfg.get('sanity_pip_check', False):
+            pip_version = det_pip_version()
+            if pip_version:
+                if LooseVersion(pip_version) >= LooseVersion('9.0.0'):
+                    run_cmd("pip check")
```

Comment:
this is non - fatal if it fails?

Reply:
It's fatal, `run_cmd` will raise an error for commands that exit with a non-zero exit code by default.